bloom/generators/debian/templates/rules.em

```diff
@@ -30,7 +30,8 @@ override_dh_auto_configure:
 	dh_auto_configure -- \
 		-DCATKIN_BUILD_BINARY_PACKAGE="1" \
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
-		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)"
+		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
+		-DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Comment:
I think I remember that now. I was going to suggest just adding the `-g` option instead of setting the `CMAKE_BUILD_TYPE`, but it seems that this is not necessary.